sd/zk/integration_test.go

```diff
@@ -17,22 +16,12 @@ var (
 	host []string
 )
 
-func TestMain(m *testing.M) {
-	flag.Parse()
-
-	fmt.Println("Starting ZooKeeper server...")
-
-	ts, err := stdzk.StartTestCluster(1, nil, nil)
-	if err != nil {
-		fmt.Printf("ZooKeeper server error: %v\n", err)
-		os.Exit(1)
+func init() {
```

Comment:
removed the code that was starting zookeeper (this previously needed the zk jar file somewhere). Not really sure the difference of init and TestMain. Would you prefer to have this code on TestMain instead?